pkgs/games/openloco/default.nix

```diff
@@ -0,0 +1,32 @@
+{ fetchFromGitHub, lib, stdenv, cmake, libpng, libzip, openal, SDL2, yaml-cpp, span-lite, gtest, pkg-config }:
+
+stdenv.mkDerivation rec {
+  pname = "openloco";
+  version = "23.02";
+
+  src = fetchFromGitHub {
+    owner = "OpenLoco";
+    repo = "OpenLoco";
+    rev = "v${version}";
+    hash = "sha256-35g7tnKez4tnTdZzavfU+X8f3btFG6EbLkU+cqL6Qek=";
+  };
+
+  # the upstream build process determines the version tag from git, since `src` is not a git checkout we just set it manually
```

Comment:
```suggestion
  # the upstream build process determines the version tag from git; since we 
  # are not using a git checkout, we should patch it manually
```